<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>330</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>562</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>562</width>
    <height>330</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>C:\Windows\System32\cmd.exe</string>
  </property>
  <property name="windowIcon">
   <iconset theme="none">
    <normaloff>../.designer/backup</normaloff>../.designer/backup</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>561</width>
      <height>331</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Pro,monospace</family>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#000;color:#fff;font-family:&quot;Source Code Pro&quot;,monospace;</string>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="backgroundVisible">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
